server/utils/nunjucksSetup.ts

```diff
@@ -20,7 +20,14 @@ export default function nunjucksSetup(app: express.Express): void {
 
   app.locals.asset_path = '/assets/'
   app.use((req, res, next) => {
-    const externalUrls = ['/link', '/barcode', '/contact-helpdesk', '/cookies-policy', '/privacy-policy']
+    const externalUrls = [
```

Comment:
We maintain an array of 'external' urls - ie. urls that our external users will use (Legal Senders)
This is because we have 1 UI app that serves both the Legal Senders and the internal mail room users. There are UI design elements such as the header content and page titles etc that need to be different based on your user type! 😢